db/migrations/20200526095904_rebuild.js

```diff
@@ -24,7 +24,7 @@ function buildOrganization(knex) {
 function buildSettings(knex) {
   return knex.schema.createTable('settings', function (table) {
     table.uuid('id').notNull().primary();
-    table.uuid('organization_id').notNull().references('organizations.id').onDelete('CASCADE');
+    table.integer('organization_id').notNull().references('organizations.id').onDelete('CASCADE');
```

Comment:
We need a field named `privacy_policy_url` to capture where the org's privacy policy is stored.

Reply:
Done